MANIFEST.in

```diff
@@ -34,6 +34,7 @@ recursive-include cernopendata *.tpl
 recursive-include cernopendata *.txt
 recursive-include cernopendata *.variables
 recursive-include cernopendata *.xml
+recursive-include elasticsearch-proxy *
```

Comment:
I had to add this otherwise `check-manifest` was failing. I recall having this issue locally but never when uploading the code to GitHub, maybe it was only checking updated files and since this PR changes many files it checked again? Just a silly guess.

Reply:
The core feature works, so approved :+1: I'm not sure about adding nginx certs to the package, but it doesn't look too great ... but then we never release on PyPI, so I guess it's OK.

Reply:
Maybe you can amend recipe to have `check-manifest --ignore ...` in `run-tests.sh`. That may be nicer than including nginx certs into Python package.

Reply:
Should `elasticsearch-proxy/.htpasswd` be included?

Reply:
I think that if we ignore certificates, the `.htpasswd` should be ignored too.

Reply:
Updated!